client/lib/signup/step-actions.js

```diff
@@ -163,7 +163,7 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 				title: siteTitle,
 			},
 		},
-		public: getNewSitePublicSetting( siteType ),
+		public: getNewSitePublicSetting( siteType, state ),
```

Comment:
So maybe:

```jsx
const isFullSiteEditingActivated = getCurrentFlowName( state ) === 'test-fse';
const isPublic = getNewSitePublicSetting( siteType ) && isFullSiteEditingActivated;
```

Another way that might be cleaner is forcing new users into the selected private by default test variant, when we enter the start/test-fse step. If we did this, would this ruin any testing results @jblz ?

<img width="315" alt="Screen Shot 2019-08-13 at 12 03 20 PM" src="https://user-images.githubusercontent.com/1270189/62969602-838ef500-bdc2-11e9-9ce2-cd1dc768301a.png">

Reply:
I'd probably just contain it to the flow as has been done here, so existing users maintain their grouping when traversing the main flow.